min_and_max.py

```diff
@@ -0,0 +1,16 @@
+numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]

```

Comment:
Here the same as for _number_bytes = int(input("Write bites: "))_

Reply:
Done